packages/builder/src/components/design/settings/controls/FieldConfiguration/PrimaryColumnFieldSetting.svelte

```diff
@@ -0,0 +1,96 @@
+<script>
```

Comment:
I just duplicated `FieldSetting` to make this, seemed better than a bit of optionally to conditionally display a tooltip imo.